### ISSUE TITLE
[Feature]: make links in DOCS navigatable with vimium/surfingkeys

### ISSUE THREAD

**Issue by molleweide** [remix-run/remix#1631]
### What is the new or updated feature that you are suggesting?

All links should be updated on the documentations website so that you can navigate links with accessibility tools such as vimium/surfingkers or some other tool. Personally I use Surfingkeys and until now I haven't found a single link in the docs that work which.

### Why should this feature be included?

The reason why this feature should be added is because I and other people who use accessibility tools don't use the mouse pointer because it generates a lot of chronic pain. This is an ergonomic issue. Not a technologically related one. If I get time to dig into the docs myself I will try to rectify this myself.

**Comment by molleweide** [comment]
It seems that no links generated from my remix project are navigatable with surfingkeys either, so I am wondering if this is an issue with surfingkeys or with remix? I'll keep this open til I figure something out.

**Comment by chaance** [comment]
@molleweide Have you found the culprit yet? All of the links we render are plain `<a>` tags that validate as expected, as long as your links have content. I can't imagine why any accessibility tool wouldn't be able to identify those.

**Comment by molleweide** [comment]
Hi!
No my apologies because I havent really delved into this so I dunno atm.
My specs were macbook with brave browser at the time of creating the issue. I haven't gotten any time to check out remix since then.

**Comment by molleweide** [comment]
allright, so I just tested this on brave again on my macbook air and it seems surfingkeys which I use now does not recognize any links on the site `https://remix.run/docs/en/v1` so the issue persists but another app I am using called Vimac does work. However, I am not sure what links vimac picks up sometimes because it is more intended for the mac OS and not the browser.

vimac fyi : https://github.com/dexterleng/vimac

**Comment by molleweide** [comment]
allright, so I just tested this on brave again on my macbook air and it seems surfingkeys which I use now does not recognize any links on the site `https://remix.run/docs/en/v1` so the issue persists but another app I am using called Vimac does work. However, I am not sure what links vimac picks up sometimes because it is more intended for the mac OS and not the browser.

vimac fyi : https://github.com/dexterleng/vimac

**Comment by machour** [comment]
@molleweide I just tried using surfingkeys on a macbook pro, with both Chrome & Brave, everything seems to be working normally, I can open link using the extension keyboard shortcuts.

<img width="1144" alt="Screen Shot 2022-09-02 at 11 57 47 PM" src="https://user-images.githubusercontent.com/304450/188244427-9b44d64c-ecee-44e0-9858-f26645b4b04d.png">


Could you maybe check again, and see if you get any errors in the console, or any pointers to the problem?

Could be also helpful to check on another computer to see if the problem still occurs, or if there's something with your specific setup, because as Chance mentioned, these are plain HTML anchor tags, and we really fail to see what is going wrong here 🤷🏼

**Comment by molleweide** [comment]
Allright seems to be something wierd on my end maybe hmm.

Fyi, i dont have time to dig into this issuea atm. 

Maybe consider close for now?

**Comment by machour** [comment]
Thank you for your feedback, closing as requested.
